bin/optimise-analytics.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+SCHEMA_VERSION=1
+
+# On developers environment export SOLR_HOST and export SOLR_COLLECTION before running
+HOST=${SOLR_HOST:-"localhost:8983"}
+COLLECTION=${SOLR_COLLECTION:-"bulk-analytics-v${SCHEMA_VERSION}"}
+
+echo "Optimising analytics..."
+# creates a new file descriptor 3 that redirects to 1 (STDOUT)
+exec 3>&1
+
+HTTP_STATUS=$(curl -w "%{http_code}" -o >(cat >&3) -s "http://${HOST}/solr/${COLLECTION}/update?optimize=true")
```

Comment:
I think default is 1 segment, but it can be modified
```
http://hostname:port/solr/<COLLECTION_NAME>/update?optimize=true&maxSegments=1' 
```
as well as the `waitFlush`:
```
optimize=true&maxSegments=10&waitFlush=false'
```

Reply:
Do we know what is the effect of these settings? I think we should understand this and improve it, possibly in another PR when we have the understanding.